<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>608</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: #8ED5F0;
    }</string>
  </property>
  <widget class="QLabel" name="sunriselabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>641</width>
     <height>191</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font-color:white;
font-size:40px;</string>
   </property>
   <property name="text">
    <string>this is the secondary window label</string>
   </property>
  </widget>
  <widget class="QLabel" name="putsunrise">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#putsunrise {
    color: white
; /* Text color */
    font-size: 25px; /* Font size */
    text-transform: uppercase; /* Makes the text uppercase */
    font-weight: bold; /* Makes the text bold */
 
   }</string>
   </property>
   <property name="text">
    <string>6:50</string>
   </property>
  </widget>
  <widget class="QLabel" name="putsunset">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#putsunset {
    color: white
; /* Text color */
    font-size: 25px; /* Font size */
    text-transform: uppercase; /* Makes the text uppercase */
    font-weight: bold; /* Makes the text bold */
 
   }</string>
   </property>
   <property name="text">
    <string>5:30</string>
   </property>
  </widget>
  <widget class="QLabel" name="suns">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#suns {
    color: white
; /* Text color */
    font-size: 25px; /* Font size */
    text-transform: uppercase; /* Makes the text uppercase */
    font-weight: bold; /* Makes the text bold */
 
   }</string>
   </property>
   <property name="text">
    <string>sunrise</string>
   </property>
  </widget>
  <widget class="QLabel" name="sunr">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#sunr {
    color: white
; /* Text color */
    font-size: 25px; /* Font size */
    text-transform: uppercase; /* Makes the text uppercase */
    font-weight: bold; /* Makes the text bold */
 
   }</string>
   </property>
   <property name="text">
    <string>sunset</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature1{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature3{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature4{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature2{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature5{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature6
{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature7
{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltemperature8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>460</y>
     <width>63</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltemperature8{
color:rgb(255, 255, 255);
font-size:25px;
}</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition1{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition5{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition2
{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>550</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition6
{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition3{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>550</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition7{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>390</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition4{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcondition8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>550</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labelcondition8{
color:rgb(255, 255, 255);
font-size:15px;
text-align:left;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>349</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime1{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>22:30</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>349</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime3{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>349</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime4{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>349</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime2{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime5{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>509</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime6{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>509</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime7{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltime8">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>509</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#labeltime8{
color:rgb(255, 255, 255);
font-size:20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
